be/src/olap/bloom_filter_predicate.h

```diff
@@ -30,15 +30,27 @@
 namespace doris {
 
 // only use in runtime filter and segment v2
+
+class BloomFilterColumnHleper : public ColumnPredicate {
+public:
+    BloomFilterColumnHleper(uint32_t column_id, int id)
```

Comment:
I think you could find a more common method to do it for all predicate not only for bloomfilter. For example, you could add a get_id() or to_string method to columnpredicate.
and you could save a map <predicate_id, filterstats> to save the stats for every predicate.
So if we want to add other filter stats for a new predicate (not bloom filter), it could be more expansible.